internal/resources/ekscluster/resource_ekscluster.go

```diff
@@ -0,0 +1,330 @@
+/*
+Copyright 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package ekscluster
+
+import (
+	"context"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/pkg/errors"
+
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/authctx"
+	clienterrors "github.com/vmware/terraform-provider-tanzu-mission-control/internal/client/errors"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/helper"
+	eksmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/ekscluster"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/common"
+)
+
+type (
+	contextMethodKey struct{}
+)
+
+var ignoredTags = []string{
+	"tmc.cloud.vmware.com/tmc-creator",
```

Comment:
can we just ignore all the tags with prefix "tmc.cloud.vmware.com" because these tags are meant to be added from service